client/src/app/shared/shared-main/video/video.service.ts

```diff
@@ -274,6 +276,16 @@ export class VideoService implements VideosProvider {
     return this.buildBaseFeedUrls(params)
   }
 
+  async getVideoSubscriptionFeedUrls (accountId: number) {
```

Comment:
I think you can add the `feedToken` as param so you don't need to use inject `scopedTokensService` in the video service